src/geometry/Line.cpp

```diff
@@ -0,0 +1,51 @@
+#include <cmath>
+
+#include "geometry/Line.h"
+
+namespace cuhksz {
+
+#define EPS 1e-6
+
+int dcmp(double x) {
+    if (std::fabs(x) < EPS) return 0;
+    return x > 0 ? 1 : -1;
+}
+
+Line::Line(Point<2> A, Vector<2> v) : A(A), v(v) { }
+
+// Line::Line(Point<2> A, Point<2> B) : A(A) {
+//     v = B - A;
+// }
+
+Line::Line(const Line& src) {
+    A = src.A;
+    v = src.v;
+}
+
+Line::~Line() { }
+
+Line& Line::operator= (const Line& src) {
+    A = src.A;
+    v = src.v;
+    return *this;
+}
+
+Point<2> getIntersection(const Line& l1, const Line& l2) {
```

Comment:
Are these 3 functions below aimed to be exported? If so, declare them in the header file.